docs/standard/base-types/regular-expressions.md

```diff
@@ -78,7 +79,7 @@ The <xref:System.String> class includes a number of string search and replacemen
   
 ### Example 2: Identify duplicated words  
 
- Accidentally duplicating words is a common error that writers make. A regular expression can be used to identify duplicated words, as the following example shows.  
+ Accidentally duplicating words is a common error that writers make. A regular expression can be used to identify duplicated words, as the following example shows:
```

Comment:
Instead of "A regular expression can be used...." can we say "Use a regular expression to identify duplicated words ...."

Reply:
Ok, updated the content.